.gitignore

```diff
@@ -1,2 +1,4 @@
 /.idea/
 /out/
+main
```

Comment:
This may not be required as the binary output goes inside the `out` directory.

Reply:
removed